app.js

```diff
@@ -0,0 +1,17 @@
+// Importing modules
+const express = require('express');
+const path = require('path');
+var favicon = require('serve-favicon')
+const app = express();
+
+app.get('/', (req, res) => {
+    res.sendFile(path.join(__dirname, '/public/home.html'));
```

Comment:
BTW you can rename home.html to index.html. In this case you don't need this block.

Webpack or Parcel can do the renaming! 😁